docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
```

Comment:
Some comments about this:
- I think we should have this `docker-compose` in a different repository so we'll be able to run many stuff like `e2e`, `prod`, `dev` environments. Does it make sense?
- Also, I've another question, is it working how does the `backend` know where is "located" the `mongodb` service? We should add some env vars to do that.